react/Header/NotificationsLink/NotificationsLink.less

```diff
@@ -0,0 +1,25 @@
+@import (reference) "~seek-style-guide/theme";
+@import (reference) "../Header.less";
+
+.bell {
+  padding: 15px 0;
```

Comment:
This value seems odd, feels like the bell should be vertically centered or have it's height calculated from rows, eg. touchable height or similar. Thoughts?